### PR TITLE
(maint) update modulesync files

### PR DESCRIPTION
**Pull request by ferventcoder**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -27,14 +27,19 @@ end
 
 group :development do
   gem 'rake',                                :require => false
-  gem 'rspec', '~>2.14.1',                   :require => false
+  gem 'rspec', '~>3.0.0',                    :require => false
```
Comment (ferventcoder) [inline Gemfile]: I'm not sure how this will play out. We may need to back this one out.

**Comment by tphoney** [comment]
@ferventcoder just in case you missed it, failing unit tests :(

**Comment by ferventcoder** [comment]
Looks like the specs need fixed up to handle the new version of rspec as well.

**Comment by mentat** [comment]
Hi there, it looks like there are still test failures.  Could you take a look?

**Comment by ferventcoder** [comment]
@mentat I'm on holiday. I also work at Puppet Labs. I think I followed up in email recently about this. ;)

**Comment by mentat** [comment]
@ferventcoder enjoy your holiday. Cheers.

**Comment by tphoney** [comment]
@ferventcoder can i help with this ? I am aware that you probably have other work to do :P

**Comment by ferventcoder** [comment]
If you would like to @tphoney, that's fine. If not, we will eventually get back to this. It's not high on the radar at the moment.

File: Gemfile [hunk 2]
```diff
@@ -25,16 +25,28 @@ def location_for(place_or_version, fake_version = nil)
...
+#if RUBY_VERSION =~ /^2/
+#  gem 'pry-byebug'
+#else
+#  gem 'pry-debugger'
+#end
+
 group :development do
   gem 'rake',                                :require => false
-  gem 'rspec', '~>2.14.1',                   :require => false
+  gem 'rspec', '~>3.0',                      :require => false
```
Comment (ferventcoder) [inline Gemfile]: Tests will need fixed up to handle new version of rspec.

**Comment by ferventcoder** [comment]
@puppetlabs/windows this should be ready in a bit for testing.

**Pull request merged**